node.test.yaml

```diff
@@ -11,4 +11,5 @@ pearl:                        # used only for 'backend_type: pearl'
   max_data_in_blob: 10000     # optional
   blob_file_name_prefix: bob  # optional
   pool_count_threads: 4       # required for 'pearl'
+  fail_retry_timeout: 100     # mls, retry to reinit pearl backend after fail. required for 'pearl'
```

Comment:
mls -> ms

Reply:
fail_retry_timeout -> fail_retry_timeout_ms - it's much better to know measurments

Reply:
nope, you can write "100sec" or "10sec 10ms"